### DISCUSSION TITLE
Can not viewer Designer Window

### DISCUSSION THREAD

**Discussion by DevX-Cipher** [icsharpcode/ILSpy#2864] [Development]
Hi i cant seem to view designer form in latest build for some reason

**Comment by christophwille** [comment]
Designer Window? What are you referring to?

**Comment by DevX-Cipher** [comment]
trying to view how you codded button for dark mode ect

**Comment by christophwille** [comment]
oh, you mean opening the ILSpy in Visual Studio and viewing the WPF Designer for the main window.

**Comment by DevX-Cipher** [comment]
yes doesn't seem to work

**Comment by siegfriedpammer** [comment]
What is the error message you are seeing? It works fine for me in VS 2022.

**Comment by siegfriedpammer** [comment]
This is the XAML used for the dark mode button.

```xml
	<CheckBox IsChecked="{Binding SessionSettings.IsDarkMode}" ToolTip="{x:Static properties:Resources.DarkMode}">
		<Image Source="Images/DarkMode.png" Stretch="None" Style="{StaticResource DarkModeAwareImageStyle}"/>
	</CheckBox>
```

In the end the dark mode handling is implemented here: https://github.com/icsharpcode/ILSpy/blob/master/ILSpy/Themes/ThemeManager.cs

I hope this helps!

**Comment by christophwille** [comment]
Note that you need VS2022 because ILSpy is using .NET 6.0.
